src/fundus/publishers/au/__init__.py

```diff
@@ -15,3 +16,13 @@ class AU(metaclass=PublisherGroup):
             Sitemap("https://www.9news.com.au/sitemap.xml", sitemap_filter=inverse(regex_filter("sitemap-content-"))),
         ],
     )
+
+    WestAustralian = Publisher(
+        name="The West Australian",
+        domain="https://thewest.com.au/",
+        parser=WestAustralianParser,
+        sources=[
+            NewsMap("https://thewest.com.au/news-sitemap.xml"),
+            Sitemap("https://thewest.com.au/sitemap.xml", reverse=True),
```

Comment:
There's also a RSS feed containing the most recent articles: https://thewest.com.au/rss